deployments/kubernetes/ci/README.MD

```diff
@@ -0,0 +1 @@
+# Kustomize CI Overlay
```

Comment:
@phudtran add some sentence to describe what this folder contains too.

Reply:
Ok, I've updated the README